### PR TITLE
refactor(template): match mahomes folder structure to foles

### PR DESCRIPTION
**Pull request by PatrickDohn**
#### Summary

- Work Type: Refactor 

#### What I did

- updated Mahomes to fit the structure of files template

#### Checklist

- [x] Branch is in format `TYPE/scope/description`
- [x] PR title is in format `TYPE(scope): description`
- [x] Commits are in semantic format
- [x] Has Summary
- [x] Has Labels

### CODE REVIEW

File: src/components/templates/MahomesTemplate/MahomesTemplate.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { StyleSheet, View } from 'react-pdf';
 import { generatePDFTemplate } from '../BaseTemplate/BaseTemplate';
-import { ContentColumn } from './ContentColumn';
+import { ContentColumn } from './content/ContentColumn';
```
Comment (davidholyko) [inline src/components/templates/MahomesTemplate/MahomesTemplate.js:5]: prob want to move Heading into its own folder

File: src/components/templates/MahomesTemplate/info/infoContact.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import { Text, View } from 'react-pdf';
+
+import { styles } from './info-styles';
+
+export const Contacts = ({ email, location }) => {
+  return (
+    <View>
+      <Text style={styles['info-heading']}>Contact</Text>
+      <Text style={styles['list-item']}>{email}</Text>
+
```
Comment (davidholyko) [inline src/components/templates/MahomesTemplate/info/infoContact.js:12]: rm blank line

File: src/components/templates/MahomesTemplate/info/infoSkills.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import React from 'react';
+import { Text, View } from 'react-pdf';
+import _ from 'lodash';
+
+import { styles } from './info-styles';
+
+export const Skills = ({ data }) => {
+  return (
+    <View>
+      <Text style={styles['section-info-second']}>Skills</Text>
```
Comment (davidholyko) [inline src/components/templates/MahomesTemplate/info/infoSkills.js:11]: this style `section-info-second` is actually the same as the one for Links and Contact. we can consolidate

File: src/example-json/patrick_dohn.json [hunk 4]
```diff
@@ -0,0 +1,132 @@
+{
```
Comment (davidholyko) [inline src/example-json/patrick_dohn.json:1]: don't commit this file. We might want to add a new gitignore rule for this

File: src/components/templates/MahomesTemplate/MahomesTemplate.js (hunk 1 above)
Comment (PatrickDohn) [reply src/components/templates/MahomesTemplate/MahomesTemplate.js:5]: All set

File: src/components/templates/MahomesTemplate/info/infoContact.js (hunk 2 above)
Comment (PatrickDohn) [reply src/components/templates/MahomesTemplate/info/infoContact.js:12]: all set

File: src/components/templates/MahomesTemplate/info/infoSkills.js (hunk 3 above)
Comment (PatrickDohn) [reply src/components/templates/MahomesTemplate/info/infoSkills.js:11]: will be taking care of naming this afternoon

File: src/example-json/patrick_dohn.json (hunk 4 above)
Comment (PatrickDohn) [reply src/example-json/patrick_dohn.json:1]: ok

**Review by PatrickDohn — COMMENTED** [review]
I responded to all comments everything looks good to go.

File: src/components/templates/MahomesTemplate/info/infoContact.js (hunk 2 above)
Comment (davidholyko) [reply src/components/templates/MahomesTemplate/info/infoContact.js:12]: it seems the blank line here still. you can resolve this conversation after the commit is pushed and this becomes outdated

**Review by davidholyko — APPROVED** [review]
LGTM

File: src/components/pages/MahomesPage.js [hunk 5]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import { MahomesTemplate } from 'src/templates/MahomesTemplate';
+import data from '../../example-json/patrick_dohn.json';
```
Comment (davidholyko) [inline src/components/pages/MahomesPage.js:3]: revert this

**Pull request merged**